CHANGELOG.rst

```diff
@@ -1,6 +1,23 @@
 Changelog
 ==========
 
+.. _2.7.0:
```

Comment:
According to https://semver.org/, I would interpret as being a major update. However, it's a slightly weird situation because this update has _no_ effect on production but breaking effect on Test. I think we could therefore justify a minor update but probably should document this somewhere so we are consistent in future.

Reply:
I definitely don't think we can guarantee we won't have to do a major release (i.e. if we did a backwards incompatible change that affects Production, we must do a major release) so would be very interested to see any docs which talk about the complexities!

Reply:
Yes! It was just a recommendation tho. I can try to find the document, must be somewhere in the Confluence sea